README.md

```diff
@@ -8,7 +8,7 @@ The Storybook (sample) pages for this library are at https://bcgov.github.io/bcr
 
 # Workflow
 
-**\*\*\* NOTE: USE NODE VERSION 12 FOR THE STEPS BELOW \*\*\***
+**\*\*\* NOTE: USE NODE VERSION 16 FOR THE STEPS BELOW \*\*\***
```

Comment:
This only impacts this project's build environment (lerna, Storybook), not the components themselves.